### DISCUSSION TITLE
What version of SSL is required for self-hosted installs?

### DISCUSSION THREAD

**Discussion by lsegal** [ruby/setup-ruby#629] [Q&A] [answered]
Under self-hosted docs, the README (vaguely) states to

> * Make sure to use the same version of libssl.

What version would that be? And, if the answer is "this depends on the Ruby version", how can a user determine which version of SSL is needed?

**Comment by MSP-Greg** [comment, marked as answer]
Loren,

See https://github.com/MSP-Greg/github-actions-ruby-info/actions/runs/10020457167, which lists information in the 'Ruby Info' step.

For future reference, the above link is the most recent Actions run, if the link is broken or the logs have disappeared, go to the most recent Actions workflow run.

**Comment by lsegal** [comment]
Oh hey there!

This saved me a ton of time digging through docker container logs. Thank you!
